src/assets/styles/other/_material-reduction.scss

```diff
@@ -863,3 +863,21 @@ body .mat-calendar-previous-button::after,
 body .mat-calendar-next-button::after {
   margin: 10px;
 }
+
+.role-missing button {
```

Comment:
Let's move this somewhere, either in a separate file or even in the component with ng-deep.